content/docs/dispensary/patients.md

```diff
@@ -95,6 +95,28 @@ Then click the `Create` button at the bottom right of the page to create the pat
 
 ![New Patient](/docs/dispensary/images/patient_creation_detail.png)
 
+### Lookup a Patient
+
+If you have followed the process above, you may see a different icon to the right of a patient's details on the patient search results screen:
+
+![Fetch patient button](/docs/dispensary/images/fetch_patient_button.png)
+
+This 'download' icon indicates that a patient matches your current search criteria, but that patient is not part of your current site. The patient has been created in another store; but don't worry! You can use the patient record in your store which would avoid creating a duplicate patient record.
```

Comment:
Wonder if we can reword this so it sounds less like a problem and more like a good thing?

```suggestion
Sometimes the record for the patient you are searching for has already be created at another site. This 'download' icon indicates that a patient matches your current search criteria, and you can download all their existing patient data to your site. 
```